components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -68,9 +68,9 @@ The ServiceNow users must have the `x_chef_automate.api` role to set up the Serv
 
 The Chef Automate Integration App has three default roles:
 
-- [Role x_chef_automate.admin]({{< relref "servicenow_integration_reference#role_x_chef_automate.admin" >}})
-- [Role x_chef_automate.user]({{< relref "servicenow_integration_reference#role_x_chef_automate.user" >}})
-- [Role x_chef_automate.api]({{< relref "servicenow_integration_reference#role_x_chef_automate.api" >}})
+- [Role x_chef_automate.admin]({{< relref "servicenow_integration_reference#role x_chef_automate.admin" >}})
+- [Role x_chef_automate.user]({{< relref "servicenow_integration_reference#role x_chef_automate.user" >}})
+- [Role x_chef_automate.api]({{< relref "servicenow_integration_reference#role x_chef_automate.api" >}})
```

Comment:
```suggestion
- [Role x_chef_automate.admin]({{< relref "servicenow_integration_reference#role-x_chef_automateadmin" >}})
- [Role x_chef_automate.user]({{< relref "servicenow_integration_reference#role-x_chef_automateuser" >}})
- [Role x_chef_automate.api]({{< relref "servicenow_integration_reference#role-x_chef_automateapi" >}})
```

Reply:
oh, for a space in hyperlinked headings, we need to insert 'hypen'?

Reply:
Ian - its not working... the hyperlinks are taking me to the Events heading.

Reply:
I didn't realize of removing 'dot'. Later, after publishing to Netlify, I could trace out. My eye for detail is bad :(